Tools/px4airframes/rcout.py

```diff
@@ -62,14 +63,28 @@ def __init__(self, groups, board, post_start=False):
                 result +=   "# %s\n" % param.GetName()
                 result +=   "if param compare SYS_AUTOSTART %s\n" % id_val
                 result +=   "then\n"
-                result +=   "\tsh %s\n" % path
+                if post_start:
```

Comment:
Can you keep the logic the same for `post_start`? The only thing that needs to be different is not to print an error if there is no .post file.

Reply:
@bkueng 
you want to keep the logic with AIRFRAME for .post files?
I did it that way so that if there's no .post file rc.autostart.post will be empty, if we'll continue the same logic the script will never be empty :
   set AIRFRAME none
   if [ ${AIRFRAME} != none ]
       sh /etc/init.d/airframes/${AIRFRAME}
   fi
   unset AIRFRAME

I tried to avoid it.

Reply:
It's a small optimization, so that reasoning works too. I slightly prefer to be consistent here, but either way is fine with me.

Reply:
I agree with @bkueng, it is better to have the same logic and same output format for both files. 
Flash-wise, the 5 lines you show should be OK.

Reply:
@bkueng  @jlecoeur 
changed it to be same logic. Can you please review again?